.papr.yml

```diff
@@ -19,9 +30,10 @@ packages:
   - openssl-devel
   - redhat-rpm-config
 
-context: 'fedora/25/atomic | origin/v3.6.0-alpha.1'
+context: 'fedora/25/atomic | origin/v3.6.0-alpha.2'
```

Comment:
I suppose not. Done in latest commit ⬇️  (squashed with the last commit). I initially did this because we used to run both v1.5.0 and v3.6.0 on each PR. When 1.5 was released, this was split out as well.